app/presenters/taxon_presenter.rb

```diff
@@ -23,7 +23,8 @@ def sections
       {
         show_section: show_section?(supergroup),
         title: section_title(supergroup),
-        documents: section_document_list(supergroup)
+        documents: section_document_list(supergroup),
+        link: section_finder_link(supergroup)
```

Comment:
I wonder if `link` should be renamed to something more specific like `finder_link` or `see_more_link`, just to make it a bit clearer when you're in the view

Reply:
I like `see_more_link`